Examples/Scripts/Python/seeding.py

```diff
@@ -104,6 +161,9 @@ def addSeeding(
     particleSmearingSigmas: ParticleSmearingSigmas = ParticleSmearingSigmas(),
     initialVarInflation: Optional[list] = None,
     seedfinderConfigArg: SeedfinderConfigArg = SeedfinderConfigArg(),
+    seedFilterConfigArg: SeedFilterConfigArg = SeedFilterConfigArg(),
+    spacePointGridConfigArg: SpacePointGridConfigArg = SpacePointGridConfigArg(),
+    seedingAlgorithmConfigArg: SeedingAlgorithmConfigArg = SeedingAlgorithmConfigArg(),
```

Comment:
These arguments also need to be defined in the `NamedTypeArgs` decorator to allow them to be specified by type rather than by position (for clarity, they are out of order in `full_chain_itk.py`) or keyword (too much repeated boilerplate). Can you add in:
```
@acts.examples.NamedTypeArgs(
...
    seedFilterConfigArg=SeedFilterConfigArg,
    spacePointGridConfigArg=SpacePointGridConfigArg,
    seedingAlgorithmConfigArg=SeedingAlgorithmConfigArg,
...
)
```